transport/src/main/java/io/netty/channel/DefaultChannelConfig.java

```diff
@@ -302,6 +319,36 @@ public ChannelConfig setRecvByteBufAllocator(RecvByteBufAllocator allocator) {
         return this;
     }
 
+    public boolean isAutoFlush() {
+        return autoFlush == 1;
+    }
+
+    public ChannelConfig setAutoFlush(boolean autoFlush) {
```

Comment:
Good enough for now

> Am 22.06.2016 um 06:40 schrieb Nitesh Kant <notifications@github.com>:
> 
> In transport/src/main/java/io/netty/channel/DefaultChannelConfig.java:
> 
> > @@ -302,6 +319,36 @@ public ChannelConfig setRecvByteBufAllocator(RecvByteBufAllocator allocator) {
> >          return this;
> >      }
> >  
> > +    public boolean isAutoFlush() {
> > +        return autoFlush == 1;
> > +    }
> > +
> > +    public ChannelConfig setAutoFlush(boolean autoFlush) {
> Thinking about it more, I think I should make both get/set methods private so that it is only accessible via .getOption()/.setOption() as we haven't added them to the ChannelConfig interface anyways. WDYT?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>